docs/data-integrations/yugabyte.mdx

```diff
@@ -0,0 +1,45 @@
+---
+title: Yugabyte
+sidebarTitle: Yugabyte
+---
+
+# YugabyteDB Handler
+
+This is the implementation of the  YugabyteDB handler for MindsDB.
+
+## YugabyteDB
+YugabyteDB is a high-performance, cloud-native distributed SQL database that aims to support all PostgreSQL features. It is best to fit for cloud-native OLTP (i.e. real-time, business-critical) applications that need absolute data correctness and require at least one of the following: scalability, high tolerance to failures, or globally-distributed deployments.
+
+## Implementation
+This handler was implemented using the `psycopg2`, a Python library that allows you to use Python code to run SQL commands on YugabyteDB.
+
+The required arguments to establish a connection are,
+* `user`: username asscociated with database
+* `password`: password to authenticate your access
+* `host`: host to server IP Address or hostname
+* `port`: port through which TCPIP connection is to be made
+* `database`: Database name to be connected
+
+
+## Usage
+In order to make use of this handler and connect to yugabyte in MindsDB, the following syntax can be used,
+~~~~sql
```

Comment:
Please correct the code blocks to be as below:

> \```sql
> SQL GOES HERE
> \```

Reply:
Not able to understand. Can you please elaborate?

Reply:
Do i need to put 3 tilde operators or make sql query single liner?

Reply:
@saif-rehman-1 
Currently, the code block you use is as below:

> \~~~~sql
> CREATE DATABASE yugabyte_datasource
> WITH
> engine='yugabyte',
> parameters={
>     "user":"admin",
>     "password":"1234",
>     "host":"127.0.0.1",
>     "port":5433,
>     "database":"yugabyte"
> };
> \~~~~

Please correct it to be as below:

> \```sql
> CREATE DATABASE yugabyte_datasource
> WITH
> engine='yugabyte',
> parameters={
>     "user":"admin",
>     "password":"1234",
>     "host":"127.0.0.1",
>     "port":5433,
>     "database":"yugabyte"
> };
> \```

The same goes for the `SELECT` query below.

Reply:
Ok Thank You. Feedback Addressed.